src/test/java/uk/ac/ebi/atlas/experimentpage/json/opentargets/EvidenceServiceIT.java

```diff
@@ -74,22 +71,41 @@ void setUp() {
 
     @Test
     void organismPartIsIncludedIfAvailable() {
-        ImmutableList.Builder<JsonObject> listBuilder = ImmutableList.builder();
-        MicroarrayExperiment experiment = (MicroarrayExperiment) experimentTrader.getPublicExperiment("E-GEOD-43049");
+        var listBuilder = ImmutableList.<JsonObject>builder();
+        var experiment = (MicroarrayExperiment) experimentTrader.getPublicExperiment("E-GEOD-40611");
         subject.evidenceForExperiment(
                 experiment, contrast -> {
                     MicroarrayRequestPreferences requestPreferences = new MicroarrayRequestPreferences();
-                    requestPreferences.setFoldChangeCutoff(0.0);
-                    requestPreferences.setCutoff(1.0);
                     requestPreferences.setHeatmapMatrixSize(1000);
                     requestPreferences.setSelectedColumnIds(ImmutableSet.of(contrast.getId()));
                     return new MicroarrayRequestContext(requestPreferences, experiment); },
                 listBuilder::add);
 
-        for (JsonObject jsonObject : listBuilder.build()) {
-            assertThat(
-                    jsonObject.get("evidence").getAsJsonObject().get("organism_part").getAsString(),
-                    is(not(emptyString())));
-        }
+        var result = listBuilder.build();
+        assertThat(result)
+                .isNotEmpty()
+                .allSatisfy(evidence ->
+                        assertThat(evidence.get("evidence").getAsJsonObject().get("organism_part").getAsString())
+                                .isNotEmpty());
+    }
+
+    @Test
+    void diseaseIdIsIncludedAsUniqueAssociation() {
+        var listBuilder = ImmutableList.<JsonObject>builder();
+        var experiment = (MicroarrayExperiment) experimentTrader.getPublicExperiment("E-GEOD-40611");
+        subject.evidenceForExperiment(
+                experiment, contrast -> {
+                    MicroarrayRequestPreferences requestPreferences = new MicroarrayRequestPreferences();
```

Comment:
missed a `var` here. `MicroarrayRequestPreferences` can be changed to `var`

Reply:
Fixed that one and a few others I missed.